README.md

```diff
@@ -209,7 +209,8 @@ Use the `can` option if you want conditionally show the menu item. This integrat
     [
         'text' => 'Add new post',
         'url' => 'admin/blog/new',
-        'can' => 'add-blog-post'
+        'can' => 'add-blog-post',
+        'model' => $model
```

Comment:
Adding this here may suggest that you can use models in the `adminlte.php` configuration file, which is not possible. I think this `'model'` documentation should be explained separately.